.buildkite/release-automation/check-nightly-commit.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
```

Comment:
can we convert the logic in this file into python? rather than using bash?

and if you use a python docker client library, you might not even need to install `docker-ce-cli`.

Reply:
done! Need to wait for this PR https://github.com/ray-project/ray/pull/44309 to be in first before this script here works